rust_src/src/fonts.rs

```diff
@@ -161,7 +161,7 @@ impl From<LispObject> for Option<LispFontObjectRef> {
     fn from(o: LispObject) -> Self {
         o.as_vectorlike().and_then(|v| {
             if v.is_pseudovector(pvec_type::PVEC_FONT) && o.is_font_object() {
-                Some(unsafe { mem::transmute(o) })
+                Some(LispFontObjectRef::new(v.as_ptr() as *mut _))
```

Comment:
Similar to my macro question/suggestion earlier. Is there a way to have "PtrRef from C pointer version"?